.github/workflows/cli-jobs-basics-hello-code.yml

```diff
@@ -17,8 +18,6 @@ jobs:
     steps:
     - name: check out repo
       uses: actions/checkout@v2
-      with:
```

Comment:
It will break some pr builds. Not sure whether we need to revert this before merging.